lint/linter/ClangFormatLinter.php

```diff
@@ -45,35 +55,78 @@ public function shouldExpectCommandErrors() {
   }
 
   protected function getMandatoryFlags() {
-    return array(
-    );
+    $options = array('-output-replacements-xml');
```

Comment:
@clementleger does this option need any specific version of clang formatter?

Reply:
We are using it with clang 6 which already includes it. Since this is a rather old version, maybe we can assume this is always there ?

Reply:
fair enough